sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -153,6 +173,16 @@ def _from_generated(cls, generated):
             )
         )
 
+    def __str__(self):
+        # type: () -> str
+        return "version={}, enabled={}, include_apis={}, retention_policy={}".format(
+            self.version, self.enabled, self.include_apis, self.retention_policy
+        )
```

Comment:
I'm thinking maybe we just don't define `__str__` for the time being. Formatting with an instance will default to using `__repr__` in the meantime anyway.

Reply:
I went with this suggestion and removed `__str__` methods for now, we can always add them back later.